Libraries/src/Amazon.Lambda.Annotations/Amazon.Lambda.Annotations.csproj

```diff
@@ -4,8 +4,13 @@
     <AssemblyVersion>1.4.0</AssemblyVersion>
     <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-  </PropertyGroup>	
-	
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Nullable>annotations</Nullable>
+    <LangVersion>10.0</LangVersion>
```

Comment:
What was the reason this was changed? I get nervous about this because the package also has to run in .NET Framework which doesn't support all language version 10 features. Unless it is required I would recommend not doing this.

Reply:
I wanted to set the `Filters` property in `SQSEventAttribute` to a be a nullable string (`string?`). However, nullable reference types are only supported in C# 8.0+. 

I have reverted this change and resorted to using a non-nullable string for `Filters` but with a default value of `null`